### PR TITLE
Update dependencies to avoid conflicts

### PR DESCRIPTION
**Pull request by elrayle**

### CODE REVIEW

File: hydra-works.gemspec [hunk 1]
```diff
@@ -12,14 +12,16 @@ Gem::Specification.new do |spec|
...
   spec.homepage      = 'https://github.com/projecthydra-labs/hydra-works'
   spec.license       = 'APACHE2'
+  spec.required_ruby_version = '>= 2.1.0'
 
   spec.files         = `git ls-files -z`.split("\x0")
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
+  spec.add_dependency 'rails', '= 4.2.7'
```
Comment (jcoyne) [inline hydra-works.gemspec]: There is no dependency on rails in this gem.

**Comment by jeremyf** [comment]
Ping @elrayle

**Comment by jcoyne** [comment]
@elrayle This PR is getting stale. Can we close it?

**Comment by elrayle** [comment]
If memory servers correctly, this was an issue for the Dive-into-Hydra-Works tutorial.  Given the comments on the PR, it seems like this PR should close with out merging.

**Pull request closed without merging**